.travis.yml

```diff
@@ -43,3 +52,13 @@ script:
   - ./configure
   - make
   - make test
+  - rm -Rf docs/dev
```

Comment:
If I understand this right, the `cache:` section above keeps docs/dev around between builds, but then you clear it out here. Why would we do both of those? Why not just start with a clear one?

Reply:
You are right. I was compiling sources on my repository from gh-pages branch and deploying artifacts to same branch. Thanks for noticing and sorry for making you look for that mistake.